caravel/views.py

```diff
@@ -1241,21 +1241,25 @@ def add_slices(self, dashboard_id):
     def testconn(self):
         """Tests a sqla connection"""
         try:
-            database_name = request.json.get('database_name')
-            database = (
-                db.session
-                .query(models.Database)
-                .filter_by(database_name=database_name)
-                .first()
-            )
             request_uri = request.json.get('uri')
```

Comment:
uri = request.json.get('uri')

so that you can save a couple of assigments later?